### PR TITLE
Fix: Init browser for ServerRuntime

### PR DESCRIPTION
**Pull request by SmartManoj**
**What is the problem that this fixes or functionality that this introduces? Does it fix any open issues?**

Solves the following one of the two errors from #3128

[FAILED tests/unit/test_runtime.py::test_simple_browse[ServerRuntime] - opendevin.core.exceptions.BrowserUnavailableException: Browser environment is not available, please check if has been initialized](https://github.com/OpenDevin/OpenDevin/actions/runs/10126129464/job/28002718521#step:8:2025) 

---

**Give a summary of what the PR does, explaining any non-trivial design decisions**

Added browser tool and used full path for python

---
**Other references**

### CODE REVIEW

File: tests/unit/test_runtime.py [hunk 1]
```diff
@@ -306,7 +306,9 @@ async def test_simple_browse(temp_dir, box_class):
     runtime = await _load_runtime(temp_dir, box_class)
 
     # Test browse
-    action_cmd = CmdRunAction(command='python -m http.server 8000 > server.log 2>&1 &')
+    action_cmd = CmdRunAction(
+        command='$OPENDEVIN_PYTHON_INTERPRETER -m http.server 8000 > server.log 2>&1 &'
```
Comment (xingyaoww) [inline tests/unit/test_runtime.py:310]: `$OPENDEVIN_PYTHON_INTERPRETER` may not be available in `EventStreamRuntime`

Comment (SmartManoj) [reply tests/unit/test_runtime.py:310]: In future? The test are passed now. Python is only not available in stream runtime

Comment (SmartManoj) [reply tests/unit/test_runtime.py:310]: image
[link](https://github.com/OpenDevin/OpenDevin/actions/runs/10138446082/job/28030667691?pr=3156#step:8:1641)

Comment (SmartManoj) [reply tests/unit/test_runtime.py]: Yes, `$OPENDEVIN_PYTHON_INTERPRETER` is not available in `EventStreamRuntime`
and `python` is not available in `ServerRuntime`

**Comment by SmartManoj** [comment]
> FAILED tests/unit/test_runtime.py::test_simple_browse[ServerRuntime] - opendevin.core.exceptions.BrowserUnavailableException: Browser environment is not available, please check if has been initialized

The mentioned error is solved. The following error will be solved by #3155 

image

[Action link](https://github.com/OpenDevin/OpenDevin/actions/runs/10160933629/job/28099301846?pr=3156#step:8:1919)

**Comment by rbren** [comment]
We're doing some cleanup of PRs, and are going to close this one for now.

**Pull request closed without merging**